### ISSUE TITLE
Amazon price checking fails after several checks

### ISSUE THREAD

**Issue by aGoelzer** [CheapTracker/CheapTracker#19]
It seems like after several requests using the same IP and User-Agent, Amazon will reject the next one. Solution: random user agent?

**Comment by bLandais** [comment]
@ChewyBomber This bug is still going on ?

**Comment by ChewyBomber** [comment]
Yep, seems to be some kind of Amazon protection against scraping.

**Comment by bLandais** [comment]
@ChewyBomber Ok, then what is the plan ? Any counter-measure ?

My first though here is to retry few random minutes later but if the bug is still here after that....
The only "true" / "proper" solution is to reverse the Amazon application to find the entry points and reproduce the exact same behavior...

**Comment by ChewyBomber** [comment]
I think the Amazon app doesn't allow either that we visit the same page 100 times in a 10-minute period. My early tests showed that changing the User-Agent could be a workaround.

**Comment by bLandais** [comment]
Great, they are not checking IP too... ?

**Comment by ChewyBomber** [comment]
I think so.
